{"description": [{"lang": "eng","value": "Libarchive version 3.2.2 is vulnerable to a out of bounds read resulting SEGFAULT"}],"assigner": "kurt@seifried.org","problem_types": [{"description": [{"lang": "eng","value": "Out of bounds read"}]}],"references": [{"url": "https://github.com/libarchive/libarchive/issues/842"}],"requester": "kamil.frankowicz@cert.pl","serial": "1","date_assigned": "2016-12-19T15:25:58.710235","date_requested": "12/17/2016 6:06:52","cve_id": "CVE-2016-1000349","notes": [{"lang": "eng","value": "https://github.com/libarchive/libarchive/issues/842"}],"data_format": "MITRE","data_version": "3.1","state": "PUBLIC","products": [{"product": [{"version": "3.2.2","product_name": "libarchive"}],"vendor_name": "libarchive"}]}